code/mob/living/carbon/human/procs/update_icon.dm

```diff
@@ -303,21 +319,50 @@
 		UpdateOverlays(null, "bloody_shoes_r")
 		UpdateOverlays(null, "wear_shoes")
 
+	// Suit
 	if (src.wear_suit)
 		wear_sanity_check(src.wear_suit)
 		src.wear_suit.wear_image.layer = src.wear_suit.wear_layer
 
 		var/wear_state = src.wear_suit.wear_state || src.wear_suit.icon_state
-		if (islist(override_states) && ("suit-[wear_state]" in override_states))
-			src.wear_suit.wear_image.icon = src.mutantrace.clothing_icon_override
-			src.wear_suit.wear_image.icon_state = "suit-[wear_state]"
+		if (islist(suit_overrides) && (wear_state in suit_overrides))
+			src.wear_suit.wear_image.icon = src.mutantrace.clothing_icon_overcoats
 		else
 			src.wear_suit.wear_image.icon = src.wear_suit.wear_image_icon
-			src.wear_suit.wear_image.icon_state = wear_state
+		src.wear_suit.wear_image.icon_state = wear_state
 
 		src.wear_suit.update_wear_image(src, src.wear_suit.wear_image.icon != src.wear_suit.wear_image_icon)
 		src.wear_suit.wear_image.color = src.wear_suit.color
 		src.wear_suit.wear_image.alpha = src.wear_suit.alpha
+
+		if (src.organHolder?.tail)
```

Comment:
This needs to be a proc, but not sure where is best to put it.

Reply:
Added as `update_tail_clothing` in [19d9f1e](https://github.com/goonstation/goonstation/pull/9773/commits/19d9f1ec4d6fdc72dc4fb93a428c9a2b2de021c1)